packages/x-scheduler/src/joy/week-view/WeekView.css

```
@@ -0,0 +1,180 @@
+@import '../styles/tokens.css';
+
+.WeekViewContainer {
+  --weekend-background: var(--mauve-1);
+  --text-strong: var(--mauve-9);
+  --text-soft: var(--mauve-5);
+  --border-color: var(--mauve-2);
+  --fixed-cell-width: 68px;
+  --hour-height: 46px;
+  --has-scroll: 0;
+}
+
+.mode-dark .WeekViewContainer {
+  --weekend-background: var(--mauve-9);
+  --text-strong: var(--mauve-5);
+  --text-soft: var(--mauve-2);
+}
+
+.WeekViewContainer {
+  display: flex;
+  width: 100%;
+  border: 1px solid var(--border-color);
+  border-radius: var(--radius-4);
+}
+
+.WeekViewRoot {
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+}
+
+.WeekViewHeader {
+  display: flex;
+  flex-direction: column;
+}
+
+.WeekViewGridRow {
+  display: grid;
+  grid-template-columns: var(--fixed-cell-width) repeat(auto-fit, minmax(0, 1fr));
+  width: 100%;
+}
+
+.WeekViewHeaderRow {
+  border-bottom: 1px solid var(--border-color);
+}
+
+.WeekViewHeader[style*='--has-scroll: 1'] .WeekViewHeaderRow {
+  overflow-y: scroll;
+  scrollbar-color: transparent transparent;
+}
+
+.WeekViewAllDayEventsRow {
+  max-height: 84px;
+
+  > *:not(:last-child) {
+    border-right: 1px solid var(--border-color);
+  }
+}
+
+.WeekViewHeader[style*='--has-scroll: 1'] .WeekViewAllDayEventsRow {
+  overflow-y: scroll;
+  scrollbar-color: var(--weekend-background) var(--weekend-background);
+}
+
+/* Webkit (Chrome, Safari, Edge) */
+.WeekViewHeader[style*='--has-scroll: 1'] .WeekViewHeaderRow::-webkit-scrollbar {
+  background: transparent;
+}
+
+.WeekViewHeader[style*='--has-scroll: 1'] .WeekViewAllDayEventsRow::-webkit-scrollbar {
+  background: var(--weekend-background);
+}
+
+.WeekViewHeaderCell {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: var(--space-2);
+  padding: 10px;
+}
+
+.WeekViewAllDayEventsHeaderCell {
+  font-size: var(--font-size-1);
+  font-style: var(--em-font-style);
+  padding: var(--space-2) 0;
+  text-align: center;
```

Small alignment suggestion - I think it would look better if the `All day` label would be vertically aligned with the time labels below 🤔 
```suggestion
  padding: var(--space-2);
  text-align: end;
```